plugins/default/lib/disk_usage.py

```diff
@@ -1,35 +1,97 @@
 #!/usr/bin/env python
 import re
 import sys
+import json
+import argparse
 from subprocess import CalledProcessError
+from collections import Counter
 
-from openshift import oc
+import openshift
 import nagios
 
 
 check_disk_cmd = ("df", "--output=target,pcent,ipcent")
 # Example output:
 # /etc/hosts      19%    8%
-check_disk_output_pattern = re.compile(r"([\w/]+)\s+(\d+)%\s+(\d+)%")
+check_disk_output_pattern = re.compile(
+    r"([A-Za-z0-9\./\-_]+)\s+(\d+)%\s+(\d+)%")
```

Comment:
I see. Haven't put enough thought on that originally.
Regexps can easily bite us, would it make sense to use a negative pattern / something less restrictive?
E.g., we could assume that paths contain anything but spaces or tabs: `[^ \t]`.

Maybe there's a better way to approach this... I wonder what `df` outputs when the mount path has spaces.
Is `df` separating the fields by just a tab char?